### ISSUE TITLE
[Bug]: Files sent from other peers are stored on the local storage, unencrypted. The DB passphrase is encoded in Base64 without any encryption in the config.

### ISSUE THREAD

**Issue by EnterTheVoid-x86** [simplex-chat/simplex-chat#3883]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Linux

### OS version

Arch Linux

### App version

5.5.4

### Current Behavior

When a user sends a file to another person, SimpleX sends the receiver the file directly from the sender, and when a file transfer completes, the file gets stored on the user's hard drive, unencrypted, and does not disappear even if the reciever delete the message with the file attached for themselves.

Another note whilst digging through the config file, I found that the database passphrase is stored directly in the configuration, encoded in Base64 with no encryption.

### Expected Behavior

Files should be encrypted properly, and decrypted and copied to RAM when needed. Files should also be deleted after the receiver chooses to delete the message for themselves.

As for the passphrase issue, I suggest you compare the inputted passphrase to the checksum of itself, and if it matches the checksum stored in the file, it allows you to go on through. 

### Steps To Reproduce

File issue:
1. Send a file to a SimpleX user
2. Wait for the file to finish sending
3. Go to ~/.local/share/simplex/simplex_v1_files on Linux to see unencrypted files

Passphrase issue:
Open ~/.config/simplex/settings.properties, copy the value of InitializationVectorDBPassphrase and decode it with a Base64 decoder.

**Comment by avently** [comment]
Are you sure that you have the option Encrypt local files enabled in settings -> Privacy & security?

**Comment by mathieujobin** [comment]
Yes, I think the assumption in a lot of cases, is if the device is compromised, the malicious user has physical access to the unlocked account, all security is futile.
